client/components/deposits-overview/index.tsx

```diff
@@ -119,6 +193,12 @@ const DepositsOverview: React.FC = () => {
 						depositsSchedule={ account.deposits_schedule }
 						showNextDepositDate={ availableFunds > 0 }
 					/>
+					<ClickTooltip
+						content={ nextDepositHelpContent }
```

Comment:
I really wanted to put the tooltip markup in `children`, [composition](https://felixgerschau.com/react-component-composition/) style:

```JSX
<ClickTooltip>
  Deposits are initiated based on the following criteria:

  <ul>
    <li>The pending period in your country</li>
    <li>(etc)</li>
  </ul>
</ClickTooltip>
```

However, our tooltip uses `children` for something else – I think it allows the client to use a different element for the button/element that user clicks to _show_ the tooltip. 

Would love to refactor our `ToolTip` and friends to support arbitrary tooltip content as `children` … though not important to do now!

Reply:
I agree!

💭 I think this component was originally based on `@wordpress/components` [tooltip](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/tooltip#children-reactreactelement), for which `children` is used for "The element to which the tooltip should anchor.".

For `@wordpress/components` popover component, however, `children` is used for "elements rendered as the popover's content." 👍